### PR TITLE
Add translations for 1016 and 992

### PR DESCRIPTION
**Pull request by xhero**

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -510,6 +521,7 @@ en:
     life_dates: Life dates
     link_type: Link type
     linkage: Linkage
+    linkage: Linkage
```
Comment (ahankinson) [inline locales/en.yml:524]: Is this a duplicate?

File: locales/it.yml [hunk 2]
```diff
@@ -510,6 +521,7 @@ it:
     licensee: Licenziatario
     life_dates: Data di nascita e di morte
     link_type: Tipo di collegamento
+    linkage: Collegamenti
```
Comment (ahankinson) [inline locales/it.yml:524]: Does the Italian need to be corrected here?

**Review by ahankinson — CHANGES_REQUESTED** [review]
'linkage' seems to be a duplicate, so this should be fixed.

**Comment by xhero** [comment]
yes, you are right it is a duplicate

**Comment by ahankinson** [comment]
Do you have to close the PR? Can't you just push an update?

**Comment by xhero** [comment]
I'm not sure, can I?

**Comment by ahankinson** [comment]
Yeah... just push commits to the `ticket-992` branch and they'll appear here.

**Review by ahankinson — APPROVED** [review]

**Comment by ahankinson** [comment]
Looks good! Thanks.

**Pull request merged**